assets/store_descriptions/en-US/strings.xml

```diff
@@ -17,7 +17,7 @@
     <important_note>Important note</important_note>
     <oh_server>You need an openHAB server for this app.</oh_server>
     <short_description>Vendor and technology agnostic open source home automation</short_description>
-    <fdroid>The builds on F-Droid have FCM and crash reporting removed and will not be able to receive push notifications from openHAB Cloud.</fdroid>
+    <fdroid>The builds on F-Droid have FCM and crash reporting removed and therefore won't be able to receive push notifications from openHAB Cloud in real time, but delayed.</fdroid>
```

Comment:
The old text was more correct, since polled notifications aren't push notifications by definition. I'd add some sentence like `Instead, notifications are polled on a regular basis in these builds.` instead.